checks/evaluation/binary_artifacts/BinaryArtifactsNotPresent.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+short: Checks that binaries are not present in the repository
+desc: This rule checks that no binaries are present in the repository
```

Comment:
Minor nit, but switching from "binaries are not present" to "no binaries are present" could be confusing (present vs not present). Better to stick with "not present" which is clearer.